Payments/Operation/01_ManageAuthorizations.md

```diff
@@ -0,0 +1,126 @@
+[!!LIST (Authorizations)](../UserInterface/01a_ListAuthorizations.md)
+
+
+# Monitor authorizations
```

Comment:
```suggestion
# Manage authorizations
```
Monitor oder Manage? Vgl. alle Dateinamen in TOC.
Ich würde bei Manage bleiben, da wir immer es so nennen. Dafür hat man auch "Check" procedure.

Reply:
Man monitort hier aber